### ISSUE TITLE
Why was ore excavator removed and where should I find the reason for these changes in the future?

### ISSUE THREAD

**Issue by ethanryan12** [FTBTeam/FTB-Academy#280]
It took me a while to find out why you removed dynamic trees, and I have no idea why excavator was removed. Please explain
Edit: there is a tip (I think 16) referencing this mod too

**Comment by aaronhowser1** [comment]
We removed Dynamic Trees because it caused enough lag to make servers unplayable, and it's not widely used enough. We actually put it to a public vote, and it was voted off.

We removed Ore Excavation because we left CurseForge. That means we have to ask every mod dev for permission to include their mods in our packs, and FunwayGuy refused. So, LatvianModder made FTB Ultimine, which serves the same function (and is better in some ways, although some features are only in 1.15).

>You can hold ` (to the left of the 1 key) while breaking a block to also break all the connected blocks.

This doesn't explicitly mention OE, and it's also the same keybind that Ultimine uses, so there's no change required.
